seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/TaskExecutionService.java

```diff
@@ -129,10 +131,21 @@ public NonCompletableFuture<TaskExecutionState> deployTask(
 
             // TODO Use classloader load the connector jars and deserialize Task
             taskGroup = nodeEngine.getSerializationService().toObject(taskImmutableInfo.getGroup());
+            return deployLocalTask(taskGroup, resultFuture);
+        } catch (Throwable t) {
+            logger.severe(ExceptionUtils.getMessage(t));
```

Comment:
Can we output logs based on job level?

Reply:
This is an optimization point. Let's do it later.  Now temporarily add the JobID to the log